### ISSUE TITLE
Transition matrix of kalman filter

### ISSUE THREAD

**Issue by Charlie715** [Kemo-Huang/BiTrack#2]
why is the parameters of transition matrix in the kalman filter set to 1?  For a CV motion model, should that set as dt ?(time difference between current frame and last frame in seconds)
![screenshot-20240702-000544](https://github.com/Kemo-Huang/BiTrack/assets/142574462/926056c6-04e7-49b4-987d-ab3117baf098)

**Comment by Kemo-Huang** [comment]
Yes you can but I think the tracking association result will be approximately the same because it is a common factor. We assume dt = 1 here.
